cloud-pubsub/deployment/pubsub-hpa.yaml

```diff
@@ -1,3 +1,18 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# [START gke_pubsub_hpa]
```

Comment:
I would replace `gke` with `kubernetes_engine`  on all these region tags - it matches the docs URL (http://cloud.google.com/kubernetes-engine/docs) and the name of this repo.

Reply:
good point, I'll make this change